style/dark-style.css

```diff
@@ -171,3 +221,53 @@ body {
 .fa:hover {
   color: #ff6b00;
 }
+
+@media screen and (min-width: 650px) {
+  .dark-section {
+    flex-direction: row-reverse;
+    justify-content: flex-end;
+  }
+
+  .dark {
+    background-image: url(../assets/bkgdesktop.png);
+    background-size: 42%;
+  }
+
+  .title-justify {
+    display: flex;
+    align-items: center;
+  }
+
```

Comment:
- [ ] The value for min-width property of the media queries should be in range of 992px - 1100px. The 650px you've used is too small. Please fix it.